js/deer-utils.js

```diff
@@ -488,6 +488,22 @@ export default {
                 console.log(logMe)
             }
         }
+    },
+
+    /**
+     * Since all Linked Data has the right to be described by multiple contexts, DEER has a special
+     * syntax to support this.  The value of an HTML Attribute DEER.CONTEXT ('deer-context' by default) 
+     * can list multiple URLs like in the example below.
+     *
+     * <span deer-context="[http://example.org/context.json][https://otherexample.org/othercontext.json]"></span>
+     *
+     * Process the value and return either a single string (URL), an array of strings (URLs), or an empty string.
+     *
+     * @param contextStringValue This is the value from the DEER.CONTEXT attribute on an HTML form.  It is a string.
+    **/
+    processContextSyntax: function(contextStringValue){
+        // Regex we are thinking contextStringValue.match(?<=\[)(.*?)(?=\])
```

Comment:
fwiw, `match()` with global flags returns an Array or null, if I recall, and we won't know if something is trimmed. That said, we return the values, so no worries. Also, we don't have a way yet to return the context to the attribute.

Reply:
Right, that's what I mean with what I said above.  I don't see where we use DEER to populate the deer-context attribute, like to process an object, learn its context, then put that value into that attribute in ` <form deer-type="Person" deer-context="" />`

Reply:
We do it for the `record` being generated, but that `record` is not the HTML object with the attribute `deer-context`.